### DISCUSSION TITLE
Shuhua

### DISCUSSION THREAD

**Discussion by uyumiy** [warengonzaga/css-text-portrait-builder#39] [Showcase]
I was being curious about the CSS Text Portrait. So I searched for a build here. Thank you so much for the builder. 

![blebel](https://user-images.githubusercontent.com/94940767/143153644-cd4e895e-f504-4c7f-b1da-31d11442a4e2.png) 

She's Shuhua from G-IDLE. Been a fan. Thanks!

**Comment by uyumiy** [comment]
Song is Somebody by Keshi. His songs are cool. You can check them out!

**Comment by warengonzaga** [comment]
Thanks for sharing! @uyumiy 🥰
